.docker/Dockerfile-build

```
@@ -6,6 +6,8 @@ WORKDIR /go/src/github.com/ory/kratos
 
 ADD go.mod go.mod
 ADD go.sum go.sum
+ADD internal/httpclient/go.mod internal/httpclient/
+ADD corp/go.mod corp/
```

This also needs the .sum file:

```suggestion
ADD internal/httpclient/go.* internal/httpclient/
ADD corp/go.* corp/
```